contracts/cwd-core/src/tests.rs

```diff
@@ -148,14 +147,6 @@ fn test_instantiate_with_n_gov_modules(n: usize) {
         }
     );
 
-    assert_eq!(
-        state.version,
-        ContractVersion {
-            contract: "crates.io:cw-core".to_string(),
```

Comment:
why remove this?

Reply:
i felt this test wasn't doing anything other than making renaming harder. :) also, it doesn't actually catch if we change the workspace level name which is what was the problem here.